apps/sscs/preview/aso/sscs-postgres.yaml

```diff
@@ -6,6 +6,12 @@ metadata:
  annotations:
   serviceoperator.azure.com/reconcile-policy: detach-on-delete
 spec:
+ tags:
```

Comment:
why do we have the generic version in apps/azureserviceoperator-system/resources/flexibleserver-postgres.yaml and then this one here?

Reply:
I think the ASO resource will need to setup in the apps/azureserviceoperator-system/resources first and then in the environement/app you want it.  Again, followed whats been done on service bus

Reply:
i don't think that's right, sscs should just add the resource to their preview kustomize file and then they will get this.

Reply:
although people probably want a bit of configuration available? i.e. database version